### ISSUE TITLE
The tests for autotime pushing take an enormous amount of time to run

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#3338]
I noticed that the unit tests are very slow in CI (and also in bci when run in Vagrant). I see that this test runs for an extreme amount of time (on the order of minutes):

https://github.com/fedora-infra/bodhi/blob/9be769b4dc53e3bd65d0beb94d6708f97c49e52d/bodhi/tests/server/scripts/test_approve_testing.py#L590-L609

The other autotime tests also take minutes to run, resulting in these new tests causing the CI to run for a very long time.

**Comment by bowlofeggs** [comment]
I don't have time to debug this right now, but it seems that these tests only go slow in network restricted environments (CI and bodhi-ci), which makes it seem like something is trying to do network connections, failing, retrying for a while, and eventually giving up (hence, the tests still pass though they take a long time).

This seems very similar to the problem I solved in https://github.com/fedora-infra/bodhi/pull/3243 and indeed, I don't see message mocking in the test I cited above. I bet we just need to mock the message sending here and in the other new tests that were recently added. We should do that anyway, because it will also assert that the correct messages are being sent which is not being checked right now.
